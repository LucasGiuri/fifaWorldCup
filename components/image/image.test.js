import React from 'react';

import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Image from './image.tsx';

describe('Image', () => {
  const arg = '../../commons/images/ARG.png';
  it('basic snapshot renders', () => {
    const b = renderer.create(<Image src={arg} />);
    const tree = b.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
