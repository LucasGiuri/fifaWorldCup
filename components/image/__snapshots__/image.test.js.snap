// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image basic snapshot renders 1`] = `
.c0 {
  width: 30px;
  border-radius: 4px;
}

<img
  className="c0"
  src="../../commons/images/ARG.png"
/>
`;
